extension/src/popup/components/signTransaction/Operations.tsx

```diff
@@ -421,6 +470,45 @@ export const Operations = ({
                   operationValue={highThreshold}
                 />
               ) : null}
+              {sponsoredId ? (
+                <KeyValueWithPublicKey
+                  operationKey="Sponsored Id"
+                  operationValue={sponsoredId}
+                />
+              ) : null}
+              {account ? (
+                <KeyValueWithPublicKey
+                  operationKey="Account"
+                  operationValue={account}
+                />
+              ) : null}
+              {source ? (
+                <KeyValueWithPublicKey
+                  operationKey="Source"
+                  operationValue={source}
+                />
+              ) : null}
+              {claimants && claimants.length
+                ? claimants.map(({ _destination }, index) => (
+                    /* eslint-disable react/no-array-index-key */
+                    <div key={`${_destination}${index}`}>
+                      {/* eslint-enable */}
+                      <SubHeaderEl>Claimant {index + 1}</SubHeaderEl>
+                      <KeyValueWithPublicKey
+                        operationKey="Destination"
+                        operationValue={_destination}
+                      />
+                      {/* TODO: Add appicable predicate UI */}
```

Comment:
I spent about a day to trying to algorithmically decode and show all the possible predicates for a Claimable Balance, but the data is shaped in a way that does not make it very easy to do in a UI (check the type for `PredicateValue`, it's pretty wild).

Definitely not impossible and something we should do in the future, but it will take a bit more time and some design input to figure out how to show this information in a usable way.